pcsx2/gui/FrameForGS.cpp

```diff
@@ -557,6 +557,7 @@ void GSFrame::OnUpdateTitle( wxTimerEvent& evt )
 #endif
 
 	double fps = wxGetApp().FpsManager.GetFramerate();
+	float per = gsRegionMode == Region_NTSC ? (fps * 100) / EmuConfig.GS.FramerateNTSC.GetWhole() : (fps * 100) / EmuConfig.GS.FrameratePAL.GetWhole(); 
```

Comment:
`GetWhole()` sounds suspiciously integer-ish. Are you sure it's returning 59.94 for NTSC?

Also, did you test that if you modify this config value at the INI file then the display reflects it correctly? (e.g. if you set `FramerateNTSC=45.5` at `PCSX2_vm.ini` and you run an NTSC game  - then it would show 100% speed when the fps is 45.5)